### DISCUSSION TITLE
Disable item roots by default?

### DISCUSSION THREAD

**Discussion by Arkensor** [Arkensor/EnfusionPersistenceFramework#9] [Polls]
Hello,

if you have a moment to spare please help me with your feedback on how you handle items on the ground in your game mode. If most people do not want to persist ground items then it would be easier to disable it in the framework and let people selectively turn them on again than to force many people to override half of the prefab base classes.

Items meaning including weapons and their attachments, clothing, backpacks, and gadgets in this context.

Thank you!

**Comment by PackJC** [comment]
If they were to persist on restart, is there a way to add the time they were dropped into the persistence? So let's say the backpack was dropped 59 minutes ago, and they are removed every hour. So one minute into the restart, the backpack would disappear. 

A lot more work, but would be interesting if this is possible to do. 

by the way, this is incredible work so far.

**Reply by Arkensor** [reply]
@PackJC this is already done for the garbage manager. That is the lifetime setting in the entity save data. If you have a custom house keeping mechanism you can see how that was done and apply that to your needs.
